README.md

```diff
@@ -133,26 +133,26 @@ On a modern Linux host run the following:
 3. Use an S3 compatible client to create a bucket in the object store
 
    ```bash
-   s3cmd mb --no-ssl --host=127.0.0.1:53390 --host-bucket=  s3://rookbucket
+   s3cmd mb --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
```

Comment:
This `AWS_ENDPOINT` env var comes from the `eval $(./rook object connection --format env-var)`?